Mage/src/main/java/mage/abilities/effects/common/continuous/LookAtFaceDownCreaturesAnyTimeEffect.java

```diff
@@ -0,0 +1,59 @@
+package mage.abilities.effects.common.continuous;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.cards.Cards;
+import mage.cards.CardsImpl;
+import mage.constants.*;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.card.FaceDownPredicate;
+import mage.game.Game;
+import mage.players.Player;
+
+/**
+ * @author notgreat
+ */
+public class LookAtFaceDownCreaturesAnyTimeEffect extends ContinuousEffectImpl {
```

Comment:
Wrong effect naming, must be `LookAtOpponentFaceDownCreaturesAnyTimeEffect`

![shot_240616_141749](https://github.com/magefree/mage/assets/8344157/beedff6d-e700-4fd4-aba5-2a3c0e805b4e)